main/tables.py

```diff
@@ -0,0 +1,11 @@
+"""Defines the Drunc Message Table for the data from the Kafka messages."""
+
+import django_tables2 as tables
+
+
+class DruncMessageTable(tables.Table):
+    """Defines a Drunc Message Table for the data from the Kafka messages."""
+
+    timestamp = tables.DateTimeColumn(format="y-m-d , H:i")
```

Comment:
The sorting of this isn't compatible with the current htmx setup so we should disable it unless it's requested as a feature.